### PR TITLE
Fix example in documentation for Painless using _source

### PR DESCRIPTION
**Pull request by jdconrad**
As the title says.

### CODE REVIEW

File: docs/reference/search/request/script-fields.asciidoc [hunk 1]
```diff
@@ -50,7 +50,7 @@ GET /_search
         },
         "script_fields" : {
             "test1" : {
-                "script" : "_source.obj1.obj2"
+                "script" : "params._source.obj1.obj2"
```
Comment (nik9000) [inline docs/reference/search/request/script-fields.asciidoc]: Ooh! If you add `// TEST[setup:twitter]` then below `// CONSOLE` on this page then we'll actually try an execute the script which might be nice.

Comment (jdconrad) [inline docs/reference/search/request/script-fields.asciidoc]: Will do... do you mind pointing me to an existing example?  Thanks in advance.

**Comment by jdconrad** [comment]
Oops!  Let this one slip through the cracks.  Going to finish it now.

File: docs/reference/search/request/script-fields.asciidoc [hunk 2]
```diff
@@ -37,9 +37,9 @@ Script fields can work on fields that are not stored (`my_field_name` in
 the above case), and allow to return custom values to be returned (the
 evaluated value of the script).
 
-Script fields can also access the actual `_source` document indexed and
-extract specific elements to be returned from it (can be an "object"
-type). Here is an example:
+Script fields can also access the actual `_source` document and
+extract specific elements to be returned from it by using `params._source`.
```
Comment (rjernst) [inline docs/reference/search/request/script-fields.asciidoc]: Can you change this to `params['_source']`?

**Review by rjernst — APPROVED** [review]
LGTM

File: docs/reference/search/request/script-fields.asciidoc (hunk 2 above)
Comment (jdconrad) [inline docs/reference/search/request/script-fields.asciidoc]: Good catch.  Fixed.

**Comment by jdconrad** [comment]
@rjernst Thanks for the review!

**Pull request merged**